static/app/views/organizationIntegrations/installedIntegration.tsx

```diff
@@ -53,6 +53,19 @@ export default class InstalledIntegration extends React.Component<Props> {
     this.props.trackIntegrationAnalytics('integrations.uninstall_completed');
   }
 
+  get integrationStatus() {
+    const {integration} = this.props;
+    // multiple status fields
+    const statusList = [integration.status, integration.organizationIntegrationStatus];
+    const notActive = statusList.filter(s => s !== 'active');
```

Comment:
```suggestion
    const notActive = statusList.some(s => s !== 'active');
```

Reply:
Actually...

Reply:
```suggestion
    const firstNotActive = statusList.first(s => s !== 'active');
```

Reply:
@evanpurkhiser You mean `.find` instead of `.first`?

Reply:
Yeah I do